src/components/work/Footer/Footer.tsx

```diff
@@ -0,0 +1,97 @@
044799 89.265339,26.7731814 C90.423883,27.7425836 91.8520849,28.2271095 93.5477555,28.2271095 C94.6712695,28.2271095 95.7170607,28.0316175 96.6847642,27.6388817 C97.6521028,27.2464963 98.4942822,26.6933029 99.2109375,25.9775499 L101.039065,27.8116014 C100.107121,28.7810036 98.9836074,29.5303896 97.6699827,30.061161 C96.3552634,30.5922827 94.8975048,30.8571429 93.297437,30.8571429 C91.4331846,30.8571429 89.7787473,30.4710636 88.3333952,29.6982045 C86.8869485,28.924995 85.7645291,27.8354248 84.9643128,26.4277421 C84.1633666,25.0207601 83.7634409,23.4165343 83.7634409,21.6175172 C83.7634409,19.8177994 84.1633666,18.2202302 84.9643128,16.8241089 C85.7645291,15.4286882 86.8818399,14.3321111 88.3151504,13.5361292 Z M89.1274084,16.3840766 C88.0104625,17.3566319 87.3437979,18.6651673 87.126685,20.3082812 L99.2350206,20.3082812 C99.1397828,18.6420445 98.5625177,17.3279037 97.5057796,16.3676105 C96.4486767,15.4066165 95.0555047,14.9262947 93.3251691,14.9262947 C91.6433646,14.9262947 90.2443543,15.4125724 89.1274084,16.3840766 Z M62.311828,5.14285714 L65.6354798,5.14285714 L65.6354798,16.5575046 C66.3267123,15.3135495 67.2746156,14.3568284 68.4773746,13.6866159 C69.6804967,13.0164034 71.0687703,12.6809345 72.6421954,12.6809345 C74.4051286,12.6809345 75.9774646,13.0878492 77.3599295,13.9016787 C78.7416684,14.7155082 79.8137322,15.8521131 80.576847,17.3114936 C81.3388727,18.7715994 81.7204301,20.4474934 81.7204301,22.3366367 C81.7204301,24.20402 81.3454075,25.8614178 80.5946361,27.3084675 C79.8445908,28.7573306 78.7830552,29.8816048 77.4132968,30.6834662 C76.0431753,31.4849649 74.4766479,31.8857143 72.7137147,31.8857143 C71.1170549,31.8857143 69.7106292,31.5509707 68.4958898,30.8807582 C67.2804243,30.2109084 66.3267123,29.2534619 65.6354798,28.0091442 L65.6354798,31.7780016 L62.311828,31.7780016 L62.311828,5.14285714 Z M69.3973237,28.1168569 C70.36229,28.6909621 71.4525059,28.9778334 72.6676084,28.9778334 C73.8827108,28.9778334 74.9729267,28.6909621 75.93753,28.1168569 C76.9032224,27.5423891 77.6536307,26.7408904 78.1905702,25.7112728 C78.7260576,24.6831059 78.9947088,23.5214767 78.9947088,22.2292867 C78.9947088,20.9613955 78.7202489,19.8128224 78.1720551,18.7832049 C77.6234982,17.7546753 76.8734529,16.9520885 75.920467,16.378346 C74.966755,15.8035155 73.8827108,15.5162816 72.6676084,15.5162816 C71.4525059,15.5162816 70.36229,15.8035155 69.3973237,16.378346 C68.4319943,16.9520885 67.6757773,17.7546753 67.1275835,18.7832049 C66.5790267,19.8128224 66.3049298,20.9613955 66.3049298,22.2292867 C66.3049298,23.5457755 66.5790267,24.7128447 67.1275835,25.7290436 C67.6757773,26.7470557 68.4319943,27.5423891 69.3973237,28.1168569 Z M10.2150538,18.2529086 L31.155914,6.17142857 L52.0967742,18.2529086 L52.0967742,36 L31.155914,23.9192498 L31.155914,12.2121686 L15.4502688,21.2732786 L15.4502688,32.9807247 L10.2150538,36 L10.2150538,18.2529086 Z M0,11.7271867 L20.4568916,0 L25.5376344,2.93294846 L5.11440194,14.6583632 L5.10938888,31.8857143 L0,28.9545378 L0,11.7271867 Z" />
```

Comment:
same here, can we make this svg a component

Reply:
👍